community-test/src/test/scala/CommunityDottySuite.scala

```diff
@@ -68,7 +68,7 @@ class CommunityDottySuite extends FunSuite {
     CommunityBuild(
       "https://github.com/scalameta/munit.git",
       // latest commit from 27.11.2020
```

Comment:
might be good to keep date synced with hash.
Or if you find it not useful maybe just removing this comment

Reply:
Fixed!